### PR TITLE
Change vegetable oil to canola oil

### PR DESCRIPTION
**Pull request by rishika-verma**

### CODE REVIEW

**Comment by rishika-verma** [comment]
Please check this out

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup canola oil
+ - 3/4 cup canola oil
```
Comment (anuraag016) [inline cake-recipe.txt]: Don't you think thats too much oil

Comment (anuraag016) [reply cake-recipe.txt]: Please check if this is right

Comment (rishika-verma) [reply cake-recipe.txt]: If you are eating the calories, might as well do it right.

Comment (anuraag016) [reply cake-recipe.txt]: Okay. Just thought this might be a typo

**Pull request merged**